doc/engineering-reference/src/simulation-models-encyclopedic-reference-002/heat-exchangers.tex

```diff
@@ -1129,15 +1138,57 @@ \subsubsection{Summary of Variable Short Time Step Response Factor Model}\label{
 
 \subsubsection{References}\label{references-2-006}
 
-Eskilson, P. 1987. Thermal Analysis of Heat Extraction Boreholes. Ph.D.~Thesis, Department of Mathematical Physics, University of Lund, Lund, Sweden.
```

Comment:
Many new references. References reformatted to have a hanging indent beginning at the second line. Looks much cleaner.